src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -966,7 +964,6 @@ namespace LowTempRadiantSystem {
         }
 
         // Obtain all of the user data related to constant flow (hydronic) low temperature radiant systems...
-        BaseNum = 0;
```

Comment:
Do not reset the BaseNum when reading the ConstantFlow or it'll override the VarFlow ones in RadSysTypes